pkg/hintrunner/hint.go

```diff
@@ -215,3 +215,47 @@ func (hint WideMul128) Execute(vm *VM.VirtualMachine) error {
 
 	return nil
 }
+
+type DebugPrint struct {
+	start ResOperander
+	end   ResOperander
+}
+
+func (hint DebugPrint) Execute(vm *VM.VirtualMachine) error {
+	var startAddr memory.MemoryAddress
+	switch v := hint.start.(type) {
+	case Deref:
+		startAddr, _ = v.deref.Get(vm)
+	default:
+		return fmt.Errorf("start has to be a pointer")
+	}
+
+	var endAddr memory.MemoryAddress
+	switch v := hint.end.(type) {
+	case Deref:
+		endAddr, _ = v.deref.Get(vm)
+	default:
+		return fmt.Errorf("end has to be a pointer")
+	}
```

Comment:
This is not right. You need to use Resolve. It will return a MemoryValue which can be either a Felt or an address. You then extract an address from both of them or error instead

Reply:
Makes sense, fixed